### ISSUE TITLE
Is Connect> Email on Class Central Monitored?

### ISSUE THREAD

**Issue by atutor** [classcentral/class-central#36]
Hey CC, I've been trying for a few weeks to get through to contact@class-central.com with no luck.

Our course listing for Digital Accessibility as a Business Practice is incorrect, and it is missing the start date (Oct 9), instead listed as self guided.  The course description is also just one sentence. There's a better description can be pulled from the Canvas.net catalog

CC Listing
https://www.class-central.com/mooc/9272/canvas-network-digital-accessibility-as-a-business-practice

Course Description
https://www.canvas.net/browse/ryersonu/courses/digital-accessibility
